Package.swift

```diff
@@ -27,14 +27,16 @@ let package = Package(
         .library(name: "SpeziSchedulerUI", targets: ["SpeziSchedulerUI"])
     ],
     dependencies: [
-        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation", from: "2.1.3"),
+//        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation", from: "2.1.3"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation", branch: "lukas/platform-env-detection"),
```

Comment:
we need to change this to a tagged release before merging this PR, and after https://github.com/StanfordSpezi/SpeziFoundation/pull/25 is merged